### DISCUSSION TITLE
Update devdocs.io with latest meteor documentation

### DISCUSSION THREAD

**Discussion by wreiske** [meteor/meteor#13157] [Feature Requests]
I use [devdocs.io](https://devdocs.io) on long flights and in air-gapped environments. The last version of meteor available there is 1.5. It would be great to see the new Meteor 3.0 documentation on there at launch.

https://github.com/freeCodeCamp/devdocs

![image](https://github.com/meteor/meteor/assets/6295044/2aecefea-54a4-4749-8b38-d7f4f5326c2d)

It looks like the "scraper" is here, https://github.com/freeCodeCamp/devdocs/blob/main/lib/docs/scrapers/meteor.rb so it may just need to run an updated scrape, or might need to change the logic a bit for the newer 3.0 documentation format.

**Comment by StorytellerCZ** [comment]
Looks interesting, can't get it running locally as my Ruby version is too low. I'll jump onto my Ubuntu upgrade and see after that. Looking at the code adding the other versions should work fine. Meteor 3 documentation might require a bit different approach though.
